cdci_data_analysis/pytest_fixtures.py

```diff
@@ -133,8 +133,8 @@ def follow_output():
     print("will stop local mail server")
     print(("child:", p.pid))
     import os, signal
-    kill_child_processes(p.pid, signal.SIGKILL)
-    os.kill(p.pid, signal.SIGKILL)
+    kill_child_processes(p.pid, signal.SIGTERM)
+    os.kill(p.pid, signal.SIGTERM)
```

Comment:
Did you manage to give it a try and reproduce the same behavior I encountered?

Reply:
what did you encounter? 
note that this is draft and I did not request review, it's not ready for it.
but you can ask of course

Reply:
It doesn't find a certain library. We can have a short call where I show you the error. I think it will be easier.

Reply:
sorry if I was unclear,  this PR is not ready to be discussed or viewed. If you want we can call after I progress to some stage.

Reply:
ah, maybe you are talking about the subprocess call for the mail server? this PR has nothing to do with that

Reply:
Yes that was my point. And yes, this PR does not cover that specific issue, but at some point (with less priority of course) I will want to show you this behavior.

Reply:
ok, sure. Maybe you can add a test with that fixture and make it xfail. and an issue?

Reply:
in any case it's not related to this one

Reply:
> ok, sure. Maybe you can add a test with that fixture and make it xfail. and an issue?

There was already a test (skipped) with that fixture. But I will mark as a xfail, makes more sense.

[https://github.com/oda-hub/dispatcher-app/blob/edef0f620710a16e665f2c6d38fc1a6f987fc1db/tests/test_job_management.py#L411-L450](https://github.com/oda-hub/dispatcher-app/blob/edef0f620710a16e665f2c6d38fc1a6f987fc1db/tests/test_job_management.py#L411-L450)

Reply:
#71